docs/interactive/interactive.ipynb

```diff
@@ -17,9 +17,102 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
I don't think we want to include these outputs with our documentation, since they're executed when the docs are built so this is just unnecessarily increasing the size of the docs...can you clear outputs for all notebooks?

Reply:
Ah, so that is why people clear the outputs of notebooks before committing them to GitHub? To make them smaller?
Cleared 👍